src/illuminatio/illuminatio_runner.py

```diff
@@ -190,55 +217,66 @@ def get_docker_network_namespace(pod_namespace, pod_name):
     export = False  # also retrieve unspecifiable fields (pod uid)
     pod = api_instance.read_namespaced_pod(pod_name, pod_namespace, pretty=pretty, exact=exact, export=export)
     pod_uid = pod.metadata.uid
-    logger.info("pod_uid: " + pod_uid)
+    LOGGER.info("pod_uid: %s", pod_uid)
     if pod_uid is None:
         raise ValueError("Failed to retrieve pod uid")
 
     client = docker.from_env()
-    logger.info("fetching pause containers")
+    LOGGER.info("fetching pause containers")
     pause_containers = client.containers.list(
         filters={"label": ["io.kubernetes.docker.type=podsandbox", "io.kubernetes.pod.uid="+pod_uid]})
     if len(pause_containers) != 1:
         raise ValueError("There should be only one pause container, found %d of them." % len(pause_containers))
     container = pause_containers[0]
-    logger.info("inspecting pause container")
+    LOGGER.info("inspecting pause container")
     inspect = client.api.inspect_container(container.id)
     net_ns = inspect.get("NetworkSettings", {}).get("SandboxKey", {})
     if not net_ns:
         raise ValueError("Could not fetch Network Namespace from Docker Runtime.")
     return net_ns
 
 
-def get_network_namespace_from(inspectp_result):
-    js = json.loads(inspectp_result)
+def extract_network_namespace(inspectp_result):
+    """
+    Extracts the network namespace information from
+    a 'crictl inspectp' result
+    """
+    json_object = json.loads(inspectp_result)
     net_ns = None
-    for ns in js["info"]["runtimeSpec"]["linux"]["namespaces"]:
-        if ns["type"] != "network":
+    for namespace in json_object["info"]["runtimeSpec"]["linux"]["namespaces"]:
+        if namespace["type"] != "network":
             continue
-        net_ns = ns["path"]
+        net_ns = namespace["path"]
         break
     return net_ns
 
 
 def get_containerd_network_namespace(host_namespace, host_name):
+    """
+    Fetches and returns the path of the network namespace
+    This function only works for runtimes that are CRI compliant e.g. containerd
```

Comment:
`This function only works for runtimes that are CRI compliant e.g. containerd` -> Why is the function then named `get_containerd_network_namespace ` and not `get_cir_network_namespace `?

Reply:
it was only tested with containerd so I was quarreling with this decision